utils/tinystr/src/ascii.rs

```diff
@@ -22,16 +22,17 @@ impl<const N: usize> TinyAsciiStr<N> {
         Self::try_from_utf8_inner(bytes, 0, bytes.len(), false)
     }
 
-    /// Creates a `TinyAsciiStr<N>` from the given utf16 byte slice.
-    /// `bytes` may contain at most `N` non-null ASCII bytes.
-    pub const fn try_from_utf16(bytes: &[u16]) -> Result<Self, TinyStrError> {
-        Self::try_from_utf16_inner(bytes, 0, bytes.len(), false)
+    /// Creates a `TinyAsciiStr<N>` from the given UTF-16 slice.
+    /// `bytes` may contain at most `N` non-null ASCII code points.
+    pub const fn try_from_utf16(code_points: &[u16]) -> Result<Self, TinyStrError> {
+        Self::try_from_utf16_inner(code_points, 0, code_points.len(), false)
     }
 
-    /// Creates a `TinyAsciiStr<N>` from a utf8 byte slice, replacing invalid bytes.
+    /// Creates a `TinyAsciiStr<N>` from a UTF-8 slice, replacing invalid code units.
     ///
-    /// Null and non-ASCII bytes (i.e. those outside the range `0x01..=0x7F`)
-    /// will be replaced with the '?' character.
+    /// Invalid code units, as well as null or non-ASCII code points
+    /// (i.e. those outside the range U+0001..=U+007F`)
+    /// will be replaced with the replacement byte.
```

Comment:
it think it makes sense to document that the replacement character is `?`

Reply:
The replacement character is an argument to the function

Reply:
oh didn't notice that change, nevermind then